tools/proofers/my_projects.php

```diff
@@ -83,6 +83,18 @@
 
 echo "<h1>" . get_usertext($page_header) . "</h1>";
 
+$can_view_post_processing = in_array("PP", $allowed_stages) or in_array("PPV", $allowed_stages);
```

Comment:
`$allowed_stages` isn't defined until line 249.

Reply:
I was testing by commenting out this line :/